pkg/controller/servicemesh/controlplane/reconciler.go

```diff
@@ -65,11 +67,17 @@ func (r *ControlPlaneReconciler) Reconcile() (reconcile.Result, error) {
 		if namespace.Labels == nil {
 			namespace.Labels = map[string]string{}
 		}
-		if label, ok := namespace.Labels["maistra.io/ignore-namespace"]; !ok || label != "ignore" {
-			r.Log.Info("Adding maistra.io/ignore-namespace=ignore label to Request.Namespace")
-			namespace.Labels["maistra.io/ignore-namespace"] = "ignore"
-			err = r.Client.Update(context.TODO(), namespace)
+
+		// Label the control plane namespace so that Kiali knows about it
+		r.Log.Info(fmt.Sprintf("Adding %s label to namespace %s", common.MemberOfKey, r.Instance.Namespace))
+		namespace.Labels[common.MemberOfKey] = r.Instance.Namespace
```

Comment:
This should be part of the test below too.  We don't want to update the namespace if we don't have to.

Reply:
Done, thanks.